cpp-inference/inference_scaling_imagenet.cpp

```diff
@@ -119,8 +119,7 @@ void run_mnist(const std::string& model_name,
   for(int i=0; i<224; i++)
```

Comment:
what is the 244?

Reply:
the char length? What happens if the char length is not fully needed, does that add extra runtime?